specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/quota.json

```diff
@@ -0,0 +1,1226 @@
+          "description": "The quota request Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the quota request.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Type of resource. \"Microsoft.Quota/ServiceLimitRequests\""
+        },
+        "properties": {
+          "description": "The quota request details.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/quotaRequestOneResourceProperties"
+        }
+      }
+    },
+    "QuotaRequestSubmitResponse": {
+      "description": "Quota submit request response",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "The quota request Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the quota request.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The quota request details.",
+          "type": "object",
+          "$ref": "#/definitions/QuotaRequestProperties"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Type of resource. \"Microsoft.Quota/quotaLimits\""
+        }
+      }
+    },
+    "QuotaRequestSubmitResponse201": {
+      "description": "The quota request submit response with request id.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The quota request id. Please use the requestId to check the request status.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The operation Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource type",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The quota request status.",
+          "$ref": "#/definitions/QuotaRequestStatusDetails",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "QuotaRequestStatusDetails": {
+      "description": "The quota request status details.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The quota request status.",
+          "$ref": "#/definitions/QuotaRequestState",
+          "readOnly": true
+        },
+        "message": {
+          "description": "A user friendly message.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "QuotaRequestDetails": {
+      "type": "object",
+      "description": "The details of the quota Request.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
```

Comment:
>"id" [](http://example.com/codeflow?start=8&length=4)

Notice that this is supposed to be a fully qualified resource identifier it cannot be any arbitrary GUID.